recipes/helios-menuinst/meta.yaml

```
@@ -0,0 +1,32 @@
+package:
+  name: helios-menuinst
+  version: "0.1"
+
+source:
+  path: src
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - mkdir -p {{ PREFIX }}/Menu
+    - cp {{ SRC_DIR }}/menu.json {{ PREFIX }}/Menu/helios.json
```

The target json file must match or start with the package name.

```suggestion
    - cp {{ SRC_DIR }}/menu.json {{ PREFIX }}/Menu/helios-menuinst.json
```